admin/sql/create_tables.sql

```diff
@@ -46,19 +46,19 @@ ALTER TABLE api_compat.session ADD CONSTRAINT session_sid_uniq UNIQUE (sid);
 
 CREATE TABLE statistics.user (
     user_id                 INTEGER NOT NULL, -- PK and FK to "user".id
-    artists                 JSONB,
-    releases                JSONB,
-    recordings              JSONB,
+    artist                  JSONB,
+    release                 JSONB,
+    recording               JSONB,
     last_updated            TIMESTAMP WITH TIME ZONE
 );
 
 CREATE TABLE statistics.artist (
     id                      SERIAL, -- PK
     msid                    UUID NOT NULL,
     name                    VARCHAR,
-    releases                JSONB,
-    recordings              JSONB,
-    users                   JSONB,
+    release                 JSONB,
+    recording               JSONB,
+    "user"                  JSONB,
```

Comment:
this is a bit annoying - I'm in favour of singular names, but user is a reserved keyword in postgres. Is there some other name we could give to this?

Reply:
What do you suggest? `listener`?

Reply:
listener sounds ok to me.